deploy/kubernetes/examples/demo-statefulset-combined.yaml

```diff
@@ -0,0 +1,40 @@
+#############$$$$$$$$############################
```

Comment:
maybe rename file to end with `-combined-vol-mode.yaml`? we may have more combinations in the future...